### PR TITLE
Jbuilder, Base and Stdio alpha1

### PR DESCRIPTION
**Pull request by diml**

### CODE REVIEW

**Comment by hcarty** [comment]
It looks like stdio references ppx_js_style which doesn't exist in the current opam repo and isn't added with this PR.

File: packages/base/base.0.1.alpha1/descr [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Full standard library replacement for OCaml
+
+Base is a complete and portable alternative to the OCaml standard
+library. It provides all standard functionalities one would expect
+from a language standard library. It uses consistent convension across
+all its module.
+
+Base aims to be usable in any context. As a result system dependant
```
Comment (avsm) [inline packages/base/base.0.1.alpha1/descr]: s/dependant/dependent

File: packages/base/base.0.1.alpha1/descr [hunk 2]
```diff
@@ -0,0 +1,12 @@
+Full standard library replacement for OCaml
+
+Base is a complete and portable alternative to the OCaml standard
+library. It provides all standard functionalities one would expect
+from a language standard library. It uses consistent convension across
```
Comment (avsm) [inline packages/base/base.0.1.alpha1/descr]: s/convension/conventions/

File: packages/stdio/stdio.0.1.alpha1/opam [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+dev-repo: "https://github.com/janestreet/stdio.git"
+license: "Apache-2.0"
+build: [
+  ["jbuilder" "build-package" "stdio" "-j" jobs]
+]
+depends: [
+  "base"
+  "jbuilder"
+  "ppx_driver"
+  "ppx_js_style"
```
Comment (avsm) [inline packages/stdio/stdio.0.1.alpha1/opam]: ppx_js_style dependency not present in this PR

**Comment by diml** [comment]
I fixed the typos and removed the ppx_js_style dependency which is not required

File: packages/base/base.0.1.alpha1/descr [hunk 4]
```diff
@@ -0,0 +1,12 @@
+Full standard library replacement for OCaml
+
+Base is a complete and portable alternative to the OCaml standard
+library. It provides all standard functionalities one would expect
+from a language standard library. It uses consistent conventions
+across all its module.
```
Comment (avsm) [inline packages/base/base.0.1.alpha1/descr]: s/all its module/all of its modules/ ?

Comment (diml) [inline packages/base/base.0.1.alpha1/descr]: Updated

**Pull request merged**